package.json

```diff
@@ -18,7 +18,7 @@
     "yaml": "^1.6.0"
   },
   "devDependencies": {
-    "electron": "^6.0.5",
+    "electron": "^5.0.10",
```

Comment:
why go back to electron 5?

Reply:
See the specific commit I made for this change. The ``dialog.showMessageBox`` in ``app.js`` in line 79 does not block the Rift Explorer process in electron 6, causing the client to be restarted immediately the moment the message box pops up without waiting for the user to click "OK". This is effectively a workaround for an error either on electron or Rift Explorer's end, but as going back one electron version fixes it I believe that to be a bug in electron.

If you find there to be a cleaner fix for this issue, I'd be happy to apply that, but for the time being I think this is the best.

Reply:
This isn't a bug. however this issue has been resolved in another PR. so downgrading isn't really necessary.

Reply:
@RayZz- Yeah, I saw that now. Apparently the ``showMessageBox`` function was renamed, and in its place a different function was created that runs asynchronously. The documentation still states that this function blocks the process though, so these two occurrences are probably the root cause of the issue here.

@Pupix Is it okay if I force-push to have just one commit remaining?

Reply:
it was renamed to `showMessageBoxSync`

Reply:
Have reverted this commit. Will leave it to your PR.